notary-eth-integration-test/src/integration-test/kotlin/integration/eth/ERC20TokenRegistrationTest.kt

```diff
@@ -57,14 +61,21 @@ class ERC20TokenRegistrationTest {
             executeTokenRegistration(tokenRegistrationConfig)
             ethTokensProvider.getTokens()
                 .map { tokensFromProvider ->
-                    val expected = tokensFromProvider
-                        .map { (ethAddress, name) ->
-                            Pair(ethAddress, EthTokenInfo(name, ethTokensProvider.getTokenPrecision(name).get()))
+                    val actual = tokensFromProvider
+                        .map { (ethAddress, tokenId) ->
+                            val name = tokenId.split("#").first()
```

Comment:
redundant splitting, we can declare a variable before it, btw can we do smth like `(first, last) = split` in Kotlin?

Reply:
Split returns list and list cannot be used in destructuring declarations.